run.sh

```diff
@@ -0,0 +1,20 @@
+set -e
+
+PBF_FILE=${1}
+
+# launch application
+docker compose down && docker compose up -d
+
+# download sample data
+wget -O data/sample.pbf ${PBF_FILE}
+
```

Comment:
downloading and running should be different steps, I find. 
maybe some (like me) has the file already downloaded then the script in its current step does not apply anymore

Reply:
If that's the case then it's probably best to just explain each step in the README, I just made a commit for that